src/modules/rlm_python/rlm_python.c

```diff
@@ -205,6 +206,58 @@ static void mod_error(void)
 	Py_XDECREF(pTraceback);
 }
 
+static int dlopen_linked_libpython_cb(struct dl_phdr_info *info,
+					UNUSED size_t size, void *data)
+{
+	const char *pattern = "/libpython"
+				STRINGIFY(PY_MAJOR_VERSION) "."
+				STRINGIFY(PY_MINOR_VERSION) ".so";
```

Comment:
Or will it by libpython2.7.dll on Cygwin and dlopen replaces the .so for us? If so, then we can drop the extension.